pt/django_start_project/README.md

```diff
@@ -83,36 +112,76 @@ DATABASES = {
-      Applying sessions.0001_initial... OK
-    
+{% filename %}command-line{% endfilename %}
+```
+(myvenv) ~/djangogirls$ python manage.py migrate
+Operations to perform:
+  Apply all migrations: auth, admin, contenttypes, sessions
+Running migrations:
+  Rendering model states... DONE
+  Applying contenttypes.0001_initial... OK
+  Applying auth.0001_initial... OK
+  Applying admin.0001_initial... OK
+  Applying admin.0002_logentry_remove_auto_add... OK
+  Applying contenttypes.0002_remove_content_type_name... OK
+  Applying auth.0002_alter_permission_name_max_length... OK
+  Applying auth.0003_alter_user_email_max_length... OK
+  Applying auth.0004_alter_user_username_opts... OK
+  Applying auth.0005_alter_user_last_login_null... OK
+  Applying auth.0006_require_contenttypes_0002... OK
+  Applying auth.0007_alter_validators_add_error_messages... OK
+  Applying sessions.0001_initial... OK
+```
 
 E está pronto! Hora de iniciar o servidor web e ver se nosso site está funcionando!
 
+## Iniciando o servidor web
+
 Você precisa estar no diretório que contém o arquivo `manage.py` (o diretório `djangogirls`). No console, nós podemos iniciar o servidor web executando o `python manage.py runserver`:
 
-    (myvenv) ~/djangogirls$ python manage.py runserver
-    
+ {% filename %}command-line{% endfilename %}
+```
+(myvenv) ~/djangogirls$ python manage.py runserver
+```
 
-Agora tudo que você precisa fazer é verificar se seu site está sendo executado - abra seu navegador (Firefox, Chrome, Safari, Internet Explorer ou o que você usa) e digite o endereço:
+Se você está em um Chromebook, use esse comando:
 
-    http://127.0.0.1:8000/
-    
+{% filename %}Cloud 9{% endfilename %}
+```
+(myvenv) ~/djangogirls$ python manage.py runserver 0.0.0.0:8080
+```
+
+Se você está em um Windows e isso falhar com um `UnicodeDecodeError`, use esse comando:
 
-O servidor web vai assumir seu prompt de comando até você pará-lo: para digitar mais comandos ou abrir uma nova janela do terminal (e não se esqueça de ativar seu virtualenv nele também), ou parar o servidor de web, alternando de volta para a janela na qual está executando e pressionando CTRL + C - botões de controle e C juntos (no Windows, você pode ter que pressionar Ctrl + Break).
+{% filename %}command-line{% endfilename %}
+```
+(myvenv) ~/djangogirls$ python manage.py runserver 0:8000
+```
+
+Agora tudo que você precisa fazer é verificar se seu site está sendo executado - abra seu navegador (Firefox, Chrome, Safari, Internet Explorer ou o que quer que seja que você usa) e digite o endereço:
+
+{% filename %}browser{% endfilename %}
+```
+http://127.0.0.1:8000/
+```
+
+Se você estiver usando um Chromebook, você sempre irá visitar seu servidor de testes acessando:
+
+{% filename %}browser{% endfilename %}
+```
+https://django-girls-<your cloud9 username>.c9users.io
+```
 
 Parabéns! Você criou seu primeiro site e o executou usando um servidor de web! Não é impressionante?
 
-![Funcionou!][3]
```

Comment:
Memo to myself: Add instructions to tutorial to change to pt-br in settings.py so the Django errors show up in Portuguese.